gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/saml/SamlSsoConfig.groovy

```diff
@@ -81,6 +85,32 @@ class SamlSsoConfig extends WebSecurityConfigurerAdapter {
 
     List<String> requiredRoles
     UserAttributeMapping userAttributeMapping = new UserAttributeMapping()
+
+    /**
+     * Ensure that the keystore exists and can be accessed with the given keyStorePassword and keyStoreAliasName
+     */
+    @PostConstruct
+    void validate() {
+      def keyStoreToValidate = keyStore
+      if (keyStoreToValidate) {
```

Comment:
For readability, I prefer to keep the nesting low, so instead of this big if block, exit early:

```
if (!keyStoreToValidate) {
  return
}
```